handlers/slots_filtered.go

```diff
@@ -103,21 +115,36 @@ func getFilteredSlotsPageData(pageIdx uint64, pageSize uint64, graffiti string,
 	return pageData, pageErr
 }
 
-func buildFilteredSlotsPageData(pageIdx uint64, pageSize uint64, graffiti string, extradata string, proposer string, pname string, withOrphaned uint8, withMissing uint8, displayColumns string) *models.SlotsFilteredPageData {
+func buildFilteredSlotsPageData(pageIdx uint64, pageSize uint64, graffiti string, invertgraffiti bool, extradata string, invertextradata bool, proposer string, pname string, invertproposer bool, withOrphaned uint8, withMissing uint8, displayColumns string) *models.SlotsFilteredPageData {
 	chainState := services.GlobalBeaconService.GetChainState()
 	filterArgs := url.Values{}
 	if graffiti != "" {
 		filterArgs.Add("f.graffiti", graffiti)
 	}
+	if invertgraffiti {
+		filterArgs.Add("f.ginvert", "on")
+	}
 	if extradata != "" {
 		filterArgs.Add("f.extra", extradata)
 	}
+	if invertextradata {
+		filterArgs.Add("f.einvert", "on")
+	}
 	if proposer != "" {
 		filterArgs.Add("f.proposer", proposer)
 	}
 	if pname != "" {
 		filterArgs.Add("f.pname", pname)
 	}
+	if invertgraffiti {
+		filterArgs.Add("f.ginvert", "on")
+	}
+	if invertextradata {
+		filterArgs.Add("f.einvert", "on")
+	}
```

Comment:
looks like `invertgraffiti` & `invertextradata` got duplicated here, so you'll find the filter twice in the generated urls ;)

Reply:
Fixed.